homeassistant/components/light/lifx/__init__.py

```diff
@@ -41,7 +45,10 @@
 
 CONF_SERVER = 'server'
 
+SERVICE_LIFX_SET_COLOR = 'lifx_set_color'
```

Comment:
Wouldn't something like `lifx_set_state` make more sense ? You're not just adjusting the color, also brightness etc.